server/channels/store/sqlstore/property_value_store.go

```diff
@@ -0,0 +1,353 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package sqlstore
+
+import (
+	"database/sql"
+	"encoding/json"
+
+	sq "github.com/mattermost/squirrel"
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/v8/channels/store"
+)
+
+var propertyValueColumns = []string{
```

Comment:
Same comment as the propertyField table.

Reply:
Fixed in https://github.com/mattermost/mattermost/commit/fdaadc04ebe7631e37cb627377abe3858c4ea623, PTAL